src/DefaultBuilder/src/WebApplication.cs

```diff
@@ -0,0 +1,243 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Hosting.Server;
+using Microsoft.AspNetCore.Hosting.Server.Features;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+// REVIEW: Or just "Microsoft.AspNetCore" like WebHost?
+namespace Microsoft.AspNetCore.Builder
```

Comment:
What do we think? `Microsoft.AspNetCore` or `Microsoft.AspNetCore.Builder`?

Reply:
Everything else is in this namespace, I don't think we have anything in the root namespace

Reply:
As I mention in the comment, `WebHost` is in the root namespace. After taking a closer look, I see that's [literally it.](https://docs.microsoft.com/en-us/dotnet/api/Microsoft.AspNetCore?view=aspnetcore-5.0) I agree that Builder is better in order to reduce `usings`.